piksi_tools/console/settings_view.py

```diff
@@ -212,6 +227,40 @@ def _factory_default_button_fired(self):
                         + "Are you sure you want to reset to factory defaults?"
     confirm_prompt.run(block=False)
 
+  def _settings_save_to_file_fired(self):
```

Comment:
Could we add something to the settings.yaml to make the baud rate non-configurable for the FTDI USART through the console by default? I feel like there's more value in making this difficult to modify and avoiding people bricking their device than there is in keeping it configurable through the console. We'll have to provide some documentation around modifying the settings through SBP messages directly at some point anyways.